File: wallet.js
class Wallet {
  constructor(balance) {
    this.balance = balance;
  }

  getBalance() {
    return this.balance;
  }

  deposit(amount) {
    this.balance += amount;
  }

  withdraw(amount) {
    if (amount > this.balance) {
      console.log("Not enough in your wallet, cannot withdraw");
    } else this.balance -= amount;
  }
}

const wallet = new Wallet(0);
module.exports = wallet;

File: wallet.py
class Wallet:
    def __init__(self):
       self.balance = 0

    def get_balance(self):
       return self.balance

    def deposit(self, amount):
       self.balance += amount

    def withdraw(self, amount):
       if amount > self.balance:
           print("Balance not enough for this withdrawal")
       else:
           self.balance -= amount